DigitalLearningSolutions.Web.Tests/ViewModels/TrackingSystem/Centre/Administrator/SearchableAdminViewModelTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.ViewModels.TrackingSystem.Centre.Administrator
+{
+    using DigitalLearningSolutions.Data.Tests.TestHelpers;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.Administrator;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    public class SearchableAdminViewModelTests
+    {
+        
+        [Test]
+        public void SearchableAdmin_centre_manager_should_not_be_able_to_deactivate_their_own_account()
+        {
+            // Given
+            var adminUser = UserTestHelper.GetDefaultAdminUser(id: 1, isCentreAdmin: true);
+            var loggedInAdminUser = UserTestHelper.GetDefaultAdminUser(id: 1, isCentreAdmin: true);
+
+            // When
+            var model = new SearchableAdminViewModel(
+                adminUser,
+                loggedInAdminUser
+            );
+
+            // Then
+            model.CanShowDeactivateAdminButton.Should().BeFalse();
+        }
+
+        public void SearchableAdmin_centre_manager_should_not_be_able_to_deactivate_other_centre_manager()
+        {
+            // Given
+            var adminUser = UserTestHelper.GetDefaultAdminUser(id: 1, isCentreAdmin: true);
+            var loggedInAdminUser = UserTestHelper.GetDefaultAdminUser(id: 1, isCentreAdmin: true);
+
+            // When
+            var model = new SearchableAdminViewModel(
+                adminUser,
+                loggedInAdminUser
+            );
+
+            // Then
+            model.CanShowDeactivateAdminButton.Should().BeFalse();
+        }
+
+        [Test]
+        public void SearchableAdmin_centre_manager_should_not_be_able_to_deactivate_super_admin_account()
+        {
+            // Given
+            var adminUser = UserTestHelper.GetDefaultAdminUser(id: 1, isCentreAdmin: true);
+            var loggedInAdminUser = UserTestHelper.GetDefaultAdminUser(id: 1, isUserAdmin: true);

```

Comment:
For full coverage the ids should be different - otherwise `CanShowDeactivateAdminButton` may be being set to false just because the ids are the same.

Reply:
Yes, different id is set.